src/AzureIoTHub.Portal/Server/Mappers/DevicePropertyProfile.cs

```diff
@@ -14,7 +15,7 @@ public DevicePropertyProfile()
             _ = CreateMap<DeviceModelProperty, DeviceProperty>();
 
             _ = CreateMap<DeviceProperty, DeviceModelProperty>()
-                .ForMember(c => c.RowKey, opts => opts.MapFrom(c => c.Name))
+                .ForMember(c => c.RowKey, opts => opts.MapFrom(c => c.Name.KeepAuthorizedCharacters()))
```

Comment:
Why not using https://github.com/CGI-FR/IoT-Hub-Portal/blob/c30edb91e69b9ab179cb67d4db2898bcd01e31cc/src/AzureIoTHub.Portal/Server/Extensions/StringExtension.cs#L23 ?